### PR TITLE
Add another-story

### PR DESCRIPTION
**Pull request by arpit731992**
"Please review this, @casey-collab"

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab Can you review this PR please?

File: english/marshmallow.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+This is a story to look for your own, to [fight your way out of this](fight-out/fight-out.md) or to calm down 
+and search for your [destiny](destiny/destiny.md).
```
Comment (casey-collab) [inline english/marshmallow.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by arpit731992** [comment]
@casey-collab Can you review this PR please?

**Comment by casey-collab** [comment]
Looks great! :hibiscus:

**Pull request merged**